src/components/Engines/Engines.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+jest.mock('./img/node.png', () => '');
+jest.mock('../Install/img/npm.svg', () => '');
+
+describe('<Engines /> component', () => {
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  test('should render the component in default state', () => {
+    const packageMeta = {
+      latest: {
+        engines: {
+          node: '>= 0.1.98',
+          npm: '>3',
+        },
+      },
+    };
+
+    jest.doMock('../../pages/version/Version', () => ({
+      DetailContextConsumer: component => {
+        return component.children({ packageMeta });
+      },
+    }));
+
+    const Engines = require('./Engines').default;
```

Comment:
Is this necessary or we can import globally with ESM import syntax?

Reply:
We need this for `jest.doMock`. This is locally scoped mock. Thanks for pointing out 😄

Reply:
@sergiohgz it is, check here https://jestjs.io/docs/en/jest-object.html#jestdomockmodulename-factory-options

It took me a while to understand this, Jest mock hoist imports, but when you want to use `jest.doMock` things are different, you need to reimport with `require` to get the mocked import properly, otherwise you get the original one from the imports hoisted above. So this configuration is correct.

Reply:
https://medium.com/@elderfo/collocating-integration-and-unit-tests-using-jest-domock-34e76cf14f6e

Reply:
All right, I understand know. But, if the png files are mocked with `identity-obj-proxy`, we can move to ESM import syntax, am i right?

Reply:
All right, I understand, as we want to mock some underlying components with `doMock`, we need this require, to load the component after mocking. Good knowledge to go bed in a while :smiley:

Reply:
Yes. I really like `doMock` API. It is super handy most of the time.